services/apps/alcs/src/portal/pdf-generation/generate-submission-document.service.ts

```diff
@@ -495,6 +495,25 @@ export class GenerateSubmissionDocumentService {
     submission: ApplicationSubmission,
     documents: ApplicationDocument[],
   ) {
+    const hasFarmStructure = submission.soilProposedStructures.some(
+      (structure) => structure.type === 'Farm Structure',
+    );
+
+    const hasResidentialStructure = submission.soilProposedStructures.some(
+      (structure) =>
+        structure.type === 'Residential - Principal Residence' ||
+        structure.type === 'Residential - Additional Residence' ||
+        structure.type === 'Residential - Accessory Structure',
+    );
```

Comment:
Wouldn't be better to compare this to an Enum like it's used here:
portal-frontend\src\app\features\notice-of-intents\edit-submission\additional-information\additional-information.component.ts
export enum STRUCTURE_TYPES {
  FARM_STRUCTURE = 'Farm Structure',
  PRINCIPAL_RESIDENCE = 'Residential - Principal Residence',
  ADDITIONAL_RESIDENCE = 'Residential - Additional Residence',
  ACCESSORY_STRUCTURE = 'Residential - Accessory Structure',
  OTHER_STRUCTURE = 'Other Structure',
}

Reply:
Yeah, good point! I was thinking there wasn't an enum like this in the services app, but there is. Probably should be applied to NOI's as well.

Reply:
I agree it would be better, but I did it this way, because a) how NOI's does it and b) there is no enum for services or an obvious place to put it. I guess I could create an enum for services. I'll just have to figure out a place that makes sense, and should probably apply to NOI's as well.